packages/cli/src/services/config-service.ts

```diff
@@ -4,18 +4,24 @@ import { exec } from 'child-process-promise'
 import { wrapExecError } from '../common/errors'
 import { checkItIsABoosterProject } from './project-checker'
 import { currentEnvironment } from './environment'
-import { pruneDevDependencies } from './dependencies'
+import { createSandboxProject, removeSandboxProject } from '../common/sandbox'
+import { installProductionDependencies } from './dependencies'
 
-type CompileAndLoadOptions = {
-  production: boolean
+export const DEPLOYMENT_SANDBOX = '.deploy'
+
+export async function createProductionSandbox(): Promise<string> {
```

Comment:
I think we should change the name of this function, right now in Booster you can define the name of your environment however you want and we should keep this code generic. Suggestion:

```suggestion
export async function createDeploymentSandbox(): Promise<string> {
```

Reply:
Would it make sense to move these sandbox management functions to their own file? It's weird that they are in `config-service.ts`

Reply:
🤣  I had that name but then I changed it! Anyway, you convinced me to use it now. Thanks!

Regarding moving these to their own file: they are! the generic version of the functions are in the file `sandbox.ts`. These functions are just wrappers that use some parameters that are specific to the scope of the `config-service` responsibility.